src/amber/router/scope.cr

```diff
@@ -1,9 +1,9 @@
 module Amber
   module Router
     class Scope
-      property :stack
+      @stack = [] of String
```

Comment:
We could simply remove this line I think.

Reply:
Sure, that's good idea. I removed.